cmd/finch/virtual_machine_status_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/runfinch/finch/pkg/dependency"
+	"github.com/runfinch/finch/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewStatusVMCommand(t *testing.T) {
+	t.Parallel()
+
+	cmd := newStatusVMCommand(nil, nil)
+	assert.Equal(t, cmd.Name(), "status")
+}
+
```

Comment:
Need one more func TestStatusVMAction_runAdapter() to improve coverage.   https://github.com/runfinch/finch/blob/d8174ff773f0f92ec94d6d97c753a872a98f74a0/cmd/finch/virtual_machine_init_test.go#L28

Reply:
Added. Please check if this is okay :)